pkgs/tools/graphics/realesrgan-ncnn-vulkan/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchzip
+, fetchFromGitHub
+, cmake
+, spirv-headers
+, vulkan-headers
+, vulkan-loader
+, glslang
+, libgcc
+, libwebp
+, ncnn
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Real-ESRGAN-ncnn-vulkan";
+  version = "0.1.3.2";
+  version_models = "0.2.3.0";
```

Comment:
```suggestion
```
There are two version numbers changing, so that isn't to useful.

Reply:
What would be a good solution for having 2 different version in one package, the 2 ideas I have are creating a second package (the models by themself wouldn't be of much use, but automatic updating would probably be easier) or just write the version number in the models url?

Reply:
Also the path for the models zip file seam random or at least not easily guessable, so that will probably stay manual work.

Reply:
I think the easiest would be if we just download the latest model and update the URL.